pkg/operator/ceph/csi/spec.go

```diff
@@ -290,17 +291,22 @@ func startDrivers(clientset kubernetes.Interface, rookclientset rookclient.Inter
 	if strings.EqualFold(enableOMAPGenerator, "true") {
 		tp.EnableOMAPGenerator = true
 	}
-	// enable RBD snapshotter by default
-	tp.EnableRBDSnapshotter = true
+	if ver.Major == KubeMinMajor && ver.Minor >= kubeMinVerForSnapshot {
```

Comment:
What if `ver.Major > KubeMinMajor` it seems like we would disable it by default instead of enable.

Reply:
Default value is just set to true when k8s >= 1.17( `ver.Major == 1  && ver.Minor >= 17 `) here.

Reply:
Ok this is fine, K8s shouldn't go to v2 anytime soon.